File: trail1/A의 배수들의 곱/product-of-multiples-of-a.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        var(A, B) = (int.Parse(arr[0]), int.Parse(arr[1]));
        int prod = 1;
        for (int i=A; i<=B; i++) {if (i % A == 0) {prod *= i;}}
        Console.WriteLine(prod);
    }
}

File: trail1/A의 배수들의 곱/product-of-multiples-of-a.cpp
#include <iostream>
using namespace std;

int main() {
    int A, B;
    cin >> A >> B;
    int prod = 1;
    for (int i=A; i<=B; i++) {if (i % A == 0) {prod *= i;}}
    cout << prod << " ";
    return 0;
}